src/database/billingdatabase.cpp

```diff
@@ -395,32 +397,30 @@ QList<Billing> BillingDatabase::getBillings(const int projectId)
     }
 
     while(q.next()) {
-        bills.append(*getBilling(q));
+        bills << *getBilling(q);
     }
+
     return bills;
+
 }
 
-QList<Billing> BillingDatabase::getAllBillingsOnly(const int idProject)
+QList<Billing> BillingDatabase::getBills(const int projectId)
 {
     QList<Billing> bills;
```

Comment:
En fait tu as utiliser la traduction de bills de français vers l'anglais et oui en français bills ça n'existe pas par contre l'inverse ça donne ça : http://www.wordreference.com/enfr/bills
Donc bon billing ça veut dire facturation, mettre un s ça voudrait dire les facturations c'est pas logique d'utiliser ce mot. Après je pense qu'on utilise plus bill pour l'addition à la fin de restau par exemple "the bill please" et que pour les factures à proprement parlé on dira plus invoice ou invoices. Enfin bref bills existe ^^